frontend/src/lib/utils/randomAnimal.ts

```diff
@@ -3,89 +3,67 @@ import { capitalizeFirstLetter, sampleOne } from '../utils'
 // Adapted from poush/random-animal
 const adjectives = [
     'furry',
-    'ferocious',
-    'dangerous',
-    'poisonous',
+    'amusing',
+    'charming',
     'tame',
-    'agile',
+    'swift',
     'clever',
-    'aggressive',
+    'pleasant',
     'tiny',
     'domestic',
     'wild',
     'herbivorous',
     'carnivorous',
     'adorable',
-    'aggressive',
     'agile',
     'beautiful',
     'bossy',
     'candid',
-    'carnivorous',
-    'clever',
     'cold',
     'colorful',
     'cuddly',
     'curious',
     'cute',
-    'dangerous',
-    'deadly',
-    'domestic',
-    'dominant',
     'energetic',
     'fast',
     'feisty',
-    'ferocious',
     'fierce',
     'fluffy',
     'friendly',
-    'furry',
     'fuzzy',
     'grumpy',
     'hairy',
     'heavy',
-    'herbivorous',
-    'jealous',
     'large',
     'lazy',
     'loud',
     'lovable',
     'loving',
     'malicious',
     'maternal',
-    'mean',
+    'sweet',
     'messy',
     'nocturnal',
     'noisy',
     'nosy',
     'picky',
     'playful',
-    'poisonous',
     'quick',
-    'rough',
     'sassy',
     'scaly',
     'short',
     'shy',
-    'slimy',
     'slow',
     'small',
     'smart',
-    'smelly',
     'soft',
-    'spikey',
-    'stinky',
     'strong',
-    'stubborn',
-    'submissive',
     'tall',
-    'tame',
     'tenacious',
     'territorial',
-    'tiny',
-    'vicious',
     'warm',
-    'wild',
+    'cheerful',
+    'gracious',
```

Comment:
if we're looking for a bit of fun, we could add team members names as adjectives here:

```suggestion
    'gracious',
    "Neil's",
    "Michael's",
    "Yakko's",
    "Tim's",
    "Marius's",
```

.. and maybe for the Hogmas people too xD ( @joethreepwood ?)
..etc.

Would be glorious to find Neil's hippopotamus 😍

Reply:
I like this idea, but it may be best to avoid adding in customer/community names. I don't want to run into issues when someone asks why 'Hannah's Beaver' or 'Dylan's Mole' or things are coming up.